contrib/pzstd/Pzstd.cpp

```diff
@@ -14,6 +14,7 @@
 #include "utils/ThreadPool.h"
 #include "utils/WorkQueue.h"
 
+#include <chrono>
```

Comment:
As a minor detail,
I like when a reason to add an include is provided in comment.
Typically, which function / type / constant is used.